__tests__/components/BB_test.js

```diff
@@ -2,11 +2,13 @@
 
 import { mount, shallow } from "enzyme";
 import React from "react";
+import configureStore from "redux-mock-store";
 
 import { BB } from "../../components/BB";
 import benefitsFixture from "../fixtures/benefits";
-import elegibilityPathsFixture from "../fixtures/eligibilityPaths";
+import eligibilityPathsFixture from "../fixtures/eligibilityPaths";
```

Comment:
whoops

Reply:
yeah it took me a while to track down that bug...